code/game/objects/items/storage/bags.dm

```diff
@@ -346,6 +346,26 @@
 	STR.insert_preposition = "in"
 	STR.can_hold = typecacheof(list(/obj/item/reagent_containers/pill, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/medspray, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/dropper))
 
+/*
+ *	Grenade bag
+ */
+
+/obj/item/storage/bag/grenade_parts
+	name = "grenade maker's bag"
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bag_grenade"
+	desc = "A bag for storing grenade components."
+	w_class = WEIGHT_CLASS_TINY
+	resistance_flags = FLAMMABLE
+
+/obj/item/storage/bag/grenade_parts/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_combined_w_class = 200
+	STR.max_items = 50
+	STR.insert_preposition = "in"
+	STR.can_hold = typecacheof(list(/obj/item/reagent_containers/glass/beaker, /obj/item/assembly/igniter, /obj/item/assembly/health, /obj/item/assembly/infra, /obj/item/assembly/mousetrap, /obj/item/assembly/prox_sensor, /obj/item/assembly/signaler, /obj/item/assembly/timer, /obj/item/assembly/voice, /obj/item/stack/cable_coil, /obj/item/screwdriver))

```

Comment:
if it isn't a subtype of `/obj/item/assembly` I'd like it removed.

If you have to refluff it to be an "assembly bag" then fine.

Reply:
Does that include screwdrivers and cables?

Reply:
yes